models/MOM6/readme.rst

```diff
@@ -9,6 +9,48 @@ Instructions for using MOM6 in CESM are available on the `MOM_interface GitHub W
 
 This DART-MOM6 interface was developed for `MOM6 <https://github.com/NCAR/MOM6>`_ within the CESM framework.
 
+MOM6 time
```

Comment:
Documentation is good! Are there other models in CESM that need this info added to their documentation pages? For example, I looked through the CLM documentation, and it doesn't talk about the calendar.

Reply:
The dart shell scripts for CLM, POP, CAM set the CALENDAR.
https://github.com/NCAR/DART/blob/1b76f3afa5978469d6a119710bb638c1c077ae20/models/clm/shell_scripts/cesm2_2/CLM5_setup_assimilation#L128
MOM6 with dart is fairly new (e.g. Robin developing the MOM6-Marble, Alper adding MOM6 multi-instance capability) so I figured it is worth noting in the docs about the calendar. 

Maybe it is worth noting about the calendar in the CESM page? @kdraeder what do you think? https://docs.dart.ucar.edu/en/latest/models/CESM/readme.html

Reply:
@hkershaw-brown we could move this discussion to email so that we can get this PR approved and released. Especially since the doc changes proposed above could be done in a separate PR. What do you think?

Reply:
@mjs2369 sounds like a plan, lets get this one out.